app/controllers/invites_controller.rb

```diff
@@ -40,12 +41,16 @@ def show
 
   def perform_accept_invitation
```

Comment:
It looks like we don't have tests for the `redeem_invite_link` flow.

Reply:
Added a test.